DAO.sol

```diff
@@ -607,7 +607,9 @@ contract DAO is DAOInterface, Token, TokenSale {
     function newContract(address _newContract){
         if (msg.sender != address(this) || !allowedRecipients[_newContract]) return;
         // move all funds
-        _newContract.call.value(address(this).balance)();
+        if (!_newContract.call.value(address(this).balance)()) {
+            return;
```

Comment:
Since nothing really happens so far in this function I think it makes no actual difference and I wanted to go with the same exit as was used 2 lines above.